src/App.js

```diff
@@ -39,15 +40,19 @@ class App extends Component {
     console.log(this.state);
   }
 
-  onItemClicked = () => {
-    // Fill this in!
+  handleItemClick = (itemType) => {//scoring based on item type clicked
+    if (itemType === 'litter') {
+      this.setState({ points: this.state.points + 1 });
+    }
   }
 
   render() {
     const items = this.state.items.map((item, i) => {
       return <GameItem
                height={item.height}     // Height - used for a CSS style to position on the screen
-               layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
+               layer={100 + i}
+               itemType={item.type} //prop for GameItem
+               onItemClick={this.handleItemClick}// 
```

Comment:
It might be a good idea to name this with the term callback, just for readability.